src/tests/subcommands/add_test.go

```diff
@@ -0,0 +1,104 @@
+package tests
+
+import (
+	"bytes"
+	"encoding/gob"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"accretional.com/semantifly/subcommands"
+)
+
+func createTempFile(t *testing.T, dir string, data []byte) *os.File {
```

Comment:
Since t is unused shouldn't we remove it as an arg?

Reply:
I am using t to give out the log and Failnow conditions for errors
```
	if err != nil {
		t.Fatalf("Failed to create temp file: %v", err)
	}
```

Reply:
Oops completely missed that, sorry